source/learn/rosetta_stone.md

```diff
@@ -0,0 +1,1513 @@
+# Python Fortran Rosetta Stone
+
+Python with NumPy and Fortran are very similar in terms of
+expressiveness and features. This rosetta stone shows how to implement
+many common idioms in both languages side by side.
+
+## How to Execute Code Snippets
+
+Consider for example the following code snippets:
+
+<table>
+<colgroup>
+<col style="width: 49%" />
+<col style="width: 50%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td>NumPy</td>
+<td><blockquote>
+<p>Fortran</p>
+</blockquote></td>
+</tr>
+<tr class="even">
+<td><pre><code>from numpy import array, size, shape, min, max, sum
+a = array([1, 2, 3])
+print shape(a)
+print size(a)
+print max(a)
+print min(a)
+print sum(a)</code></pre></td>
+<td><pre class="fortran"><code>integer :: a(3)
+a = [1, 2, 3]
+print *, shape(a)
+print *, size(a)
+print *, maxval(a)
+print *, minval(a)
+print *, sum(a)</code></pre></td>
+</tr>
+</tbody>
+</table>
+
+In Python, just save the code to a file `a.py` and execute using
```

Comment:
I think `a.py` and `a.f90` are bad choices for the name here. Especially for newcomers it might not be clear that `a.out` is not obtained by using the basename of the Fortran source file. Rather we should call them `example.py` or `test.f90`.

Reply:
Sure Sir, actually I had been re-writing the entire document in myst , so will also add this. Thanks.